config/csm/rest_response_data.yaml

```diff
@@ -58,6 +58,7 @@
     - "Invalid request body: {'role': ['Must be one of: admin, manage, monitor.']}"
     - "Invalid request body: {'role': ['Must be one of: admin, manage, monitor.'], 'password': [\"Password Policy Not Met. Must be more than 8 characters. Must contain at least one Uppercase Alphabet. Must contain at least one Lowercase Alphabet. Must include !@#$%^&*()_+-=[]{}|'.\"]}"
     - "Invalid request body: {'password': [\"Password Policy Not Met. Must be more than 8 characters. Must contain at least one Uppercase Alphabet. Must contain at least one Lowercase Alphabet. Must include !@#$%^&*()_+-=[]{}|'.\"], 'role': ['Must be one of: admin, manage, monitor.']}"
+    - "$A: Must be greater than or equal to 1."
```

Comment:
Can u use more readable variable name instead of A. I think Rajesh has used it everywhere and it action item on him aswell.

Reply:
Used readable string for message particular to my test